packages/components/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid-1-2 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 2rem;
+}
+
+.grid-1-3 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    gap: 0.625rem;
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-1-4 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    grid-template-rows: auto;
+    gap: 0.625rem;
+}
+
+@media (max-width:500px) {
+    .grid-1-2 {
+        width: 100%;
+        height: auto;
+        display: grid;
+        grid-template-columns: auto;
+        grid-template-rows: 1fr 1fr;
+        gap: 2rem;
```

Comment:
When applying media queries, we don't need to redeclare those properties that are not changing.
So you remove width, height display, gap